cdserver/app/models/unit.rb

```diff
@@ -0,0 +1,6 @@
+# A room within a shelter building, especially for a family.   Units have more
+# than one bed, but those beds are not individually tracked within CARES.
```

Comment:
Yes. The most obvious thing is to use the `.include` method to eager-load associations.